### ISSUE TITLE
Existing RPM file will be used even though new file is available

### ISSUE THREAD

**Issue by opoplawski** [opensearch-project/documentation-website#637]
NOTE: More information in comments about why this should be a documented issue. 

Original title:
[BUG] opensearch-dashboards 2.0.0-1 rpm fails to start

**Describe the bug**

opensearch-dashboards fails to start:
```
opensearch-dashboards[186714]: {"type":"log","@timestamp":"2022-06-03T17:16:38Z","tags":["error","savedobjects-service"],"pid":186714,"message":"This version of OpenSearch Dashboards (v1.3.1) is incompatible with the following OpenSearch nodes in your cluster: v2.0.0 @ os-sea01.nwra.com/10.30.10.27:9200 (10.30.10.27)"}
```

**To Reproduce**
1. Install opensearch-dashboards-2.0.0-1.x86_64 from repository
2. Start service
3. systemctl status opensearch-dashboards

**OpenSearch Version**
opensearch-2.0.0-1.x86_64

**Dashboards Version**
opensearch-dashboards-2.0.0-1.x86_64

**Host/Environment (please complete the following information):**
 - OS: AlmaLinux 8.6

**Additional context**

I tried to ask in forums here: https://forum.opensearch.org/t/opensearch-dashboards-rpm-seems-to-have-an-incorrect-version-number/9741 but got no response.

**Comment by hdhalter** [comment]
Hi @kavilla - Do you know if this is still an issue and is this the appropriate place for the update? https://opensearch.org/docs/1.3/install-and-configure/install-opensearch/rpm/. Thanks!

**Comment by kavilla** [comment]
I dont think it will hurt to mention to backup and clean the file out if blocked. I see other sections mention "this will fail if..."

**Comment by hdhalter** [comment]
@kavilla - Can you please update the topic with the information? We don't have an FAQ for installation. We only have this topic with general information: https://opensearch.org/docs/2.11/install-and-configure/install-opensearch/index/, so wherever you think is appropriate is fine. Thanks!

**Comment by natebower** [comment]
Closing as stale
